src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationDetailsPage.test.tsx

```diff
@@ -303,6 +303,33 @@ describe('PublicationDetailsPage', () => {
       });
     });
 
+    test('shows validation errors when there is no summary', async () => {
+      renderPage(testPublication);
+
+      await waitFor(() => {
+        expect(screen.getByText('Publication details')).toBeInTheDocument();
+      });
+
+      userEvent.click(
+        screen.getByRole('button', { name: 'Edit publication details' }),
+      );
+
+      await waitFor(() => {
+        expect(screen.getByLabelText('Publication title')).toBeInTheDocument();
```

Comment:
Not sure if this is a mistake or intentional, but should it be 'Publication summary'?

Reply:
I've changed it in on PublicationDetailsPage.test and PublicationForm.test.